Source/BugsnagPluginClient.m

```diff
@@ -0,0 +1,31 @@
+//
+//  BugsnagPluginClient.m
+//  Bugsnag
+//
+//  Created by Jamie Lynch on 12/03/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+#import "BugsnagPluginClient.h"
+#import "BugsnagPlugin.h"
+
+@interface BugsnagPluginClient ()
+@property NSSet<id<BugsnagPlugin>> *plugins;
+@end
+
+@implementation BugsnagPluginClient
+
+- (instancetype _Nonnull)initWithPlugins:(NSMutableSet<id<BugsnagPlugin>> *_Nonnull)plugins {
+    if (self = [super init]) {
+        _plugins = [NSSet setWithSet:plugins];
+    }
+    return self;
+}
+
+- (void)loadPlugins {
+    for (id<BugsnagPlugin> plugin in self.plugins) {
+        [plugin load];
```

Comment:
Can we catch any exception here to prevent it affecting the client start? (This is what we do in Android.)

Reply:
Updated this to catch exceptions thrown when loading a plugin, although this is less likely compared to Android as exceptions are generally avoided in Objective-C.